hdt-lib/src/triples/BitmapTriplesIterators.cpp

```diff
@@ -935,6 +940,13 @@ bool ObjectIndexIterator::isSorted(TripleComponentRole role) {
     throw std::runtime_error("Order not supported");
 }
 
+void ObjectIndexIterator::skip(unsigned int pos) {
```

Comment:
I think this method should just delegate to `goTo`; they are nearly identical. Also a candidate for inlining.